lib/src/dom/pretty_dom.dart

```diff
@@ -98,4 +101,13 @@ class PrettyDomOptions {
   /// {@endtemplate}
   external bool? get min;
   external set min(bool? value);
+
+  /// {@template prettyDomOptionsFilterNode}
+  /// ### [filterNode]
+  /// By default, `<style />`, `<script />` and comment nodes are ignored.
+  /// You can configure this behavior by passing a custom `filterNode` function that should return true for every
+  /// node that you wish to include in the output.
+  /// {@endtemplate}
+  external bool Function(Node)? get filterNode;
```

Comment:
Could we add a basic test for this option in `pretty_dom_test.dart`?

Reply:
Glad you asked because it didn't work 😄 fixed to work around https://github.com/testing-library/dom-testing-library/issues/1360!